docs/topics/request-response.rst

```diff
@@ -841,6 +841,18 @@ HtmlResponse objects
 
 .. _meta http-equiv: https://www.w3schools.com/TAGS/att_meta_http_equiv.asp
 
+JsonResponse objects
+--------------------
+
+.. class:: JsonResponse(url[, ...])
+
+    The :class:`JsonResponse` class is a subclass of :class:`TextResponse` which
+    adds a :func:`json` function that decodes the JSON body and returns
```

Comment:
```suggestion
    adds a :func:`json` method that decodes the JSON body and returns
```

Reply:
Have you rendered the documentation locally? (`tox -e docs`) Are all links working as expected?

Reply:
Yes, I have used `make html` and `make htmlview` in the "docs" folder as the Readme says. To me it looks good.